media/css/firefox/new/common/thanks-messaging.scss

```diff
@@ -71,14 +82,13 @@ $image-path: '/media/protocol/img';
     }
 
     &.linux {
-        .show-else,
-        .show-auto-download-notification {
+        .show-linux,
+        {
             display: block !important;
         }
 
         &.arm {
-            .show-else,
-            .show-auto-download-notification {
+            .show-else {
                 display: none !important;
             }
 
```

Comment:
Issue: we should remove this `linux.arm` class altogether, since it now results in double messaging. 

We should probably remove the `arm` specific strings from the `/thanks/` page. And if we wanted to go a step further, we could remove the `arm` class detection from `site.js` too?

![image](https://github.com/mozilla/bedrock/assets/400117/57d49855-aad8-4278-b938-c77b6905d386)